.github/workflows/python-app.yml

```
@@ -17,7 +17,7 @@ jobs:
       matrix:
         # Default builds are on Ubuntu
         os: [ubuntu-latest]
-        python-version: [3.7, 3.8, 3.9, pypy-3.7]
+        python-version: [3.7, 3.8, 3.9]
```

I recently learned that these should be strings. When you add Python 3.10 to the list it will have to be a string.

```suggestion
        python-version: ['3.7', '3.8', '3.9']
```